types/fs-extra-promise/index.d.ts

```diff
@@ -71,6 +71,7 @@ export function readAsync(fd: number, buffer: Buffer, offset: number, length: nu
 export function readFileAsync(filename: string, options: string | ReadOptions): Promise<string>;
 export function readFileAsync(filename: string): Promise<Buffer>;
 export function writeFileAsync(filename: string, data: any, options?: string | WriteOptions): Promise<void>;
+export function mkdtempAsync(prefix: string, options: string | {encoding: string}): Promise<string>;
```

Comment:
[nits] 'options' should be optional, like: 'options?'
But someone who wants to use as optional can send PR, so I don't think this is a big problem :+1: